app/Jobs/Network/SyncProviderNetworksJob.php

```
@@ -0,0 +1,72 @@
+    {
+        return $this->timeout + 60;
+    }
+
+    public function handle(): void
+    {
+        $this->run("provider-networks-{$this->project->id}", function (): void {
+            app(SyncProviderNetworks::class)->forProject($this->project, $this->network);
+        });
+    }
+
+    public function failed(Exception $e): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $e. Configured minimum length is 3. (undefined)

(ShortVariable)